### PR TITLE
Update Squareline Studio example to LVGL 9.1

### PR DESCRIPTION
**Pull request by aliphys**
As described in https://github.com/arduino-libraries/Arduino_GigaDisplay/issues/29 , SquareLine Studio now:
- Supports LVGL9.1
- Allows for export of the `library/UI` files

This PR modifies the existing [squarelinestudio_lvgl_v8.ino](https://github.com/arduino-libraries/Arduino_GigaDisplay/blob/de47486471e46d0c6f9d0e5fc3395407910ad44e/examples/lvgl/squarelinestudio_lvgl_v8/squarelinestudio_lvgl_v8.ino) sketch, in line with the updates made to SquareLine Studio to undo regressions. 🌟 

Closes https://github.com/arduino-libraries/Arduino_GigaDisplay/issues/29

### CODE REVIEW

**Comment by aliphys** [comment]
When compiling the [squarelinestudio_lvgl.ino](https://github.com/arduino-libraries/Arduino_GigaDisplay/blob/d9d24d7ca4b16cfd037a5de676af61a5a2fdd6e9/examples/lvgl/squarelinestudio_lvgl/squarelinestudio_lvgl.ino) sketch new `libraries/UI` folder from Squareline Studio with LVGL9.1, there is an undefined reference to `ui_init()`.

```
C:\Users\Ali Jahangiri\AppData\Local\Temp\arduino\sketches\13EED99990BB5BE14B654F6C77474EA5\sketch\objs.a(squarelinestudio_lvgl.ino.cpp.o): In function `setup':
C:\GitHub\Arduino\Arduino_GigaDisplay\examples\lvgl\squarelinestudio_lvgl/squarelinestudio_lvgl.ino:37: undefined reference to `ui_init'
collect2.exe: error: ld returned 1 exit status

Using library Arduino_H7_Video at version 1.0 in folder: C:\Users\Ali Jahangiri\AppData\Local\Arduino15\packages\arduino\hardware\mbed_giga\4.1.3\libraries\Arduino_H7_Video 
Using library Arduino_GigaDisplayTouch at version 1.0.1 in folder: C:\Users\Ali Jahangiri\Documents\Arduino\libraries\Arduino_GigaDisplayTouch 
Using library Wire in folder: C:\Users\Ali Jahangiri\AppData\Local\Arduino15\packages\arduino\hardware\mbed_giga\4.1.3\libraries\Wire (legacy)
Using library lvgl at version 9.1.0 in folder: C:\Users\Ali Jahangiri\Documents\Arduino\libraries\lvgl 
Using library Portenta_SDRAM at version 1.0 in folder: C:\Users\Ali Jahangiri\AppData\Local\Arduino15\packages\arduino\hardware\mbed_giga\4.1.3\libraries\Portenta_SDRAM 
Using library ea_malloc in folder: C:\Users\Ali Jahangiri\AppData\Local\Arduino15\packages\arduino\hardware\mbed_giga\4.1.3\libraries\ea_malloc (legacy)
exit status 1

Compilation error: exit status 1
```

I see that the following line is not changed between the previous version:
- https://github.com/arduino-libraries/Arduino_GigaDisplay/blob/de47486471e46d0c6f9d0e5fc3395407910ad44e/examples/lvgl/squarelinestudio_lvgl_v8/ui/src/ui.h#L37
- https://github.com/arduino-libraries/Arduino_GigaDisplay/blob/d9d24d7ca4b16cfd037a5de676af61a5a2fdd6e9/examples/lvgl/squarelinestudio_lvgl/ui/src/ui.h#L44

@leonardocavagnis can you advise on this?

File: examples/lvgl/squarelinestudio_lvgl/squarelinestudio_lvgl.ino [hunk 1]
```diff
@@ -23,16 +23,12 @@
 #include "Arduino_GigaDisplayTouch.h"
 
 #include "lvgl.h"
-#include "ui.h"
+#include "ui/ui.h"
```
Comment (leonardocavagnis) [inline examples/lvgl/squarelinestudio_lvgl/squarelinestudio_lvgl.ino:26]:
```suggestion
#include "ui.h"
```

**Comment by aliphys** [comment]
Following recommendations from @leonardocavagnis , the example now works. The `UI` folder must be located within the `Arduino/libraries` folder of the local installation, not the sketch itself. 
image

**Comment by aliphys** [comment]
Example also works with LVGL 8.4 (so should not break existing code created previously )
image

**Comment by aliphys** [comment]
When comparing compilation time between LVGL8.4 and LVGL9.1: **LVGL9,1 is about 60% slower**.

| LVGL Version | Compile 1 (m:s) | Compile 2 (m:s) | Compile 3 (m:s) |
|-|-|-|-|
|8.4|1:41|1:40|1:41|
|9.1|2:41|2:38|2:34|

Between each run, the library cache was emptied by switching the board as described by @per1234 [here](https://forum.arduino.cc/t/cache-clear-option/1040106/7).

Could possibly be improved by https://github.com/arduino/arduino-cli/pull/2625

Does not cause any breaking changes. So we can continue with the PR.

File: examples/lvgl/squarelinestudio_lvgl/squarelinestudio_lvgl.ino [hunk 2]
```diff
@@ -8,7 +8,7 @@
  * 1. Create a SquareLine Studio project with the following settings:
  *    - Resolution: 800x480
  *    - Color depth: 16-bit
- *    - LVGL version: 8.3.x (NOTE: It only supports LVGL version 8 and earlier.)
+ *    - LVGL version: 9.1
```
Comment (leonardocavagnis) [inline examples/lvgl/squarelinestudio_lvgl/squarelinestudio_lvgl.ino:11]:
```suggestion
 *    - LVGL version: 9.x.x
```

**Review by leonardocavagnis — COMMENTED** [review]
Everything looks good!
Thanks Ali for the help!

**Pull request merged**